zeppelin-server/src/main/java/org/apache/zeppelin/rest/InterpreterRestApi.java

```diff
@@ -243,17 +244,26 @@ public Response addRepository(String message) {
   /**
    * get the metainfo property value
    */
-  @GET
+  @POST
```

Comment:
POST is for request that would make state changes on server side, GET is just for read-only operation.  IIUC this api didn't change anything in server side, then we should use GET.